conn2res/connectivity.py

```diff
@@ -17,20 +17,53 @@
 
 class Conn:
     """
-    Class that represents a connectivity matrix representing either weighted
-    or unweighted connectivity data
+    Class that represents a weighted or unweighted network using connectivity
+    data
 
-    Attributes
-    ----------
-    # TODO
+    Notes:
+    1. The diagonal of the connectivity matrix is set to zero as well as 
+    inf and nan values are replaced to zero.
+
+    2. We makes sure that all nodes are connected to the rest of the network,
+    otherwise the network is reduced to the largest component. Importantly,
+    the original indexes of nodes is kept during this process, i.e., for
+    instance, node 95 stays node 95 even if node 90 is removed.
+
+    3. The input and output nodes should be set such that they belong to the
+    largest component, otherwise the signal cannot propagate from/to them.
 
-    Methods
+    4. Symmetric networks are checked for connectedness only in a weak sense,
+    i.e., using a network where the directed edges are replaced with
+    symmetric edges.
```

Comment:
This is the revised documentation about network connectedness.

Reply:
Thank you so much again Agoston!